plugins/optimization-detective/storage/rest-api.php

```diff
@@ -46,6 +46,11 @@ function od_register_endpoint(): void {
 			'required'    => true,
 			'pattern'     => '^[0-9a-f]{32}$',
 		),
+		'eTag'                => array(
+			'type'        => 'string',
+			'description' => __( 'ETag for the current environment.', 'optimization-detective' ),
+			'required'    => true,
```

Comment:
If using hashes for ETags, then this should have a specific `pattern`, same as is used for the `slug` argument above:

```suggestion
			'required'    => true,
			'pattern'     => '^[0-9a-f]{32}$',
```

Might as well add a `minLength` and `maxLength` to all of these hash params set to `32`.